algebra/primefield/src/prime_field.rs

```diff
@@ -434,6 +434,21 @@ where
     }
 }
 
+// TODO: Find a way to create generic implementations of these
+impl<P: Parameters<UInt = U256>> From<PrimeField<P>> for U256 {
```

Comment:
From the docs:

> Only implement Into if a conversion to a type outside the current crate is required. From cannot do these type of conversions because of Rust's orphaning rules. See Into for more details.
> - <https://doc.rust-lang.org/stable/std/convert/trait.From.html>

So in this case the recommended action is to provide a generic implementation of `Into` without `From`.